pkg/elastic/elastic_search_api_service.go

```diff
@@ -29,7 +29,7 @@ type SearchService interface {
 	GetMetaEngineName() string
 }
 
-type elasticSearchService struct {
+type ElasticSearchService struct {
```

Comment:
Why'd we make this public?

Reply:
this client is for the enterprise app search, so you won't want to use it.
When you create the new client for the regular ES, don't put it in the pkg folder though, put it in infrastructure